index.html

```diff
@@ -1195,6 +1195,16 @@ <h2>Query</h2>
         <pre class="example nohighlight">
 did:example:123456?versionId=1
         </pre>
+
+        <p class="note" title="Avoid comparison of DID URLs containing
+          multiple query parameters">
+Implementers are urged to avoid using [=DID URLs=] that contain more than a
+single query parameter when comparing [=DID URLs=] for equivalence. There are
+no normalization rules defined for query parameters in this specification and
+any query parameter normalization rules defined at the DID Method specification
+or application layer are not universally recognized rules.
```

Comment:
I could see another specification handling this for some ecosystem/use case in the future, so I have a slight suggestion. I also had some trouble following what the advice was here, so let me know if the additional rewording helps:

```suggestion
Implementers are urged to avoid comparing [=DID URLs=] with more than
one query parameter for equivalence without a specification specifically
designed for that purpose. There are no normalization rules defined for
query parameters in this specification and any query parameter normalization
rules defined at the DID Method specification or application layer are not
universally recognized rules.
```

Reply:
A light tweak to @dlongley's suggestion...

```suggestion
Implementers are urged to avoid comparing [=DID URLs=] for equivalence when
they have more than one query parameter without a specification specifically
designed for that purpose. This specification defines no normalization rules
for query parameters, and any query parameter normalization rules defined at
the DID Method specification or application layer are not universally
recognized rules.
```